remove mention of building blocks and blocks from documentation (#1660)
open-contracting/standard: docs/guidance/map/milestones.md

## Planning

> The planning milestones object is used for two types of milestones:

Reviewer: `planning.milestones` is an array, but since this section is about the purpose of planning milestones rather than their representation in data or the schema, I think it's fine to just omit the type. Also, rewrite to use active voice:

```suggestion
Planning milestones describe:
```

 * Key events in the planning process, for example, the preparation of an environmental impact assessment, the approval to proceed with a project, or the date of a public consultation. 
 * Anticipated milestones during the contract implementation stage, for example, the date by which goods delivery of the goods is required.

If during the planning process you have information about tender process milestones, then you

> populate the tender milestones object instead.

Reviewer: Also need to remove the unnecessary linebreak.

```suggestion
ought to publish it as a tender milestone.
```

## Tender

> The tender milestones object is used to describe two types of milestone:

Reviewer: See comment on planning milestones.

```suggestion
Tender milestones describe:
```

  * Key dates in the tender and award stages which are not covered by other fields, for example, the date by which procuring entity will respond to enquiries.
  * Anticipated milestones during the contract implementation stage, for example, the date by which goods need to be delivered.

## Contract

The contract milestones object is used to describe:

>   * Events related to the signing of the contract, for example, the date of commercial close in a PPP contract.

Reviewer: See comment on planning milestones. Also remove unnecessary list formatting.

```suggestion
Contract milestones describe events related to the signing of the contract, for example, the date of commercial close in a PPP contract.
```

 
## Contract Implementation

The contract implementation milestones object is used to describe:

>   * Any events related to the delivery of the contract, for example, the agreed date by which goods will be delivered.

Reviewer: See comment on contract milestones.

```suggestion
Contract implementation milestones describe events related to the delivery of the contract, for example, the agreed date by which goods will be delivered.
```

## Planning

Planning milestones describe:
 * Key events in the planning process, for example, the preparation of an environmental impact assessment, the approval to proceed with a project, or the date of a public consultation. 
 * Anticipated milestones during the contract implementation stage, for example, the date by which goods delivery of the goods is required.

If during the planning process you have information about tender process milestones, then you ought to publish it as a tender milestone.

## Tender

Tender milestones describe:

>   * Key dates in the tender and award stages which are not covered by other fields, for example, the date by which procuring entity will respond to enquiries.

Reviewer: ```suggestion
  * Key dates in the tender and award stages which are not covered by other fields, for example, the date by which the procuring entity will respond to enquiries.
```
